### ISSUE TITLE
baseNetwork and Siamese modelCheckpoints used in train script

### ISSUE THREAD

**Issue by artem-tok** [Ant-Brain/EfficientWord-Net#44]
Hello! In script https://github.com/Ant-Brain/EfficientWord-Net/blob/main/training.ipynb you are referring to 
`baseNetwork = tf.keras.models.load_model("/content/drive/MyDrive/Siamese/modelCheckpoints_old/model-8-01-0.96.h5").get_layer("basemodel")`
and 
```
model = tf.keras.models.load_model(
    "/content/drive/MyDrive/Siamese/modelCheckpoints/model-4-15-loss_0.072_val_acc0.947.h5",
    custom_objects={"triplet_loss":triplet_loss})
```
on your drive. Could you please share this models?

**Comment by TheSeriousProgrammer** [comment]
That model is old and very poor interms of accuracy, consider using the latest resnet-arcloss based ones its built with pytorch

**Comment by TheSeriousProgrammer** [comment]
Pytorch Training Repo https://github.com/Ant-Brain/EfficientWord-Net-Trainer, simple_trainer.py is the entry point

**Comment by artem-tok** [comment]
@TheSeriousProgrammer many thanks!)
